buildspec.yml

```diff
@@ -0,0 +1,24 @@
+version: 0.2
+
+env:
+  variables:
+    # WARNING! you _must_ set this in the codebuild GUI.
+    BUCKET_PREFIX: ""
+    # this is set to the latest available planet if you don't override it in the GUI!
+    PLANET_DATE: ""
+
+phases:
+  install:
+    commands:
+      - pip install pipenv
```

Comment:
If you use the Python base Docker image in codebuild I think pipenv is already installed for you.

Reply:
I thought so too (I'm using the base image called `aws/codebuild/python:2.7.12`), but on my third try (after two failed attempts to make a valid `buildspec.yml`) I got the error: ```/codebuild/output/tmp/script.sh: pipenv: not found```

Do you think it might be already installed in a non-`$PATH` location? Or perhaps installed by default on Python >=3 images?